docs/variant-specific/pink.mdx

```diff
@@ -253,7 +256,24 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
 
 <BubblegumFinesse />
 
-- _Bubblegum Finesses_ can only be used to get pink cards.
```

Comment:
why is this line deleted

Reply:
it's replaced with the line below it. I thought they were identical, with the second one being clearer to someone who is learning. 

given your comment on that line as well, it seems that may not be the case?

how do you interpret this (deleted) line?
"Bubblegum Finesses can only be used to get pink cards."

what is it actually saying?
I think the only thing it _can_ be saying is that once a pink card plays, it's a Finesse.

Reply:
hmm... did we accidentally remove the other statement?

I think it's important to communicate somewhere that Bubblegum moves causing pink cards to blind play are finesses.  That might now be missing from the doc? or is it elsewhere?

Reply:
(oh, we do have "- _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Bubblegum Finesse_." under Bubblegum Bluffs... but hmmm, is that clear enough?)

Reply:
i suspect so